contracts/ronin/RoninGovernanceAdmin.sol

```diff
@@ -15,17 +16,24 @@ contract RoninGovernanceAdmin is
   BOsGovernanceProposal,
   HasValidatorContract
 {
+  using ErrorHandler for bool;
   using Proposal for Proposal.ProposalDetail;
   using IsolatedGovernance for IsolatedGovernance.Vote;
 
   /// @dev Mapping from request hash => emergency poll
   mapping(bytes32 => IsolatedGovernance.Vote) internal _emergencyExitPoll;
 
   modifier onlyGovernor() {
-    require(_getWeight(msg.sender) > 0, "RoninGovernanceAdmin: sender is not governor");
+    _onlyGovernor();
     _;
   }
 
+  function _onlyGovernor() private view {
+    if (_getWeight(msg.sender) == 0) {
+      revert ErrUnauthorized(msg.sig);
```

Comment:
`ErrNotGovenor`

or `ErrUnauthorized(msg.sig, <string>)`, where string could be < 4 bytes, such as `GN` for denoting for Governor. `GV` for Governing Validator, etc.

Reply:
updated.